mettle/configure.ac

```diff
@@ -19,17 +19,17 @@ case $host_os in
 		HOST_OS=apple
 		OBJCFLAGS="$OBJCFLAGS -fobjc-arc"
 		AC_DEFINE(HAVE_WEBCAM)
-		AC_SUBST([PLATFORM_LDADD], ['-framework CoreMedia -framework CoreImage -framework AVFoundation -framework ImageIO'])
+		AC_SUBST([PLATFORM_LDADD], ['-framework CoreMedia -framework QuartzCore -framework AVFoundation -framework ImageIO'])
 		AC_EGREP_CPP(yes,
 			[#import <TargetConditionals.h>
-			 #if TARGET_OS_OSX
+			 #ifndef TARGET_OS_OSX
 			 yes
 			 #endif],
```

Comment:
Note sure, it should be in config.log at any rate.

For reference, we're building in a boxcutter 10.8 vagrant vm. I can upload it to an S3 bucket if you want to replicate.